### PR TITLE
change IndustrialDigitalOutSubIDs

### PR DESCRIPTION
**Pull request by Rosi2143**
add check for
* OUT1_VALUE
* OUT2_VALUE
* OUT3_VALUE
to IndustrialDigitalOutSubIDs

### CODE REVIEW

**Comment by 9037568** [comment]
This change fails to compile. Also, from what I can see, the file being modified here is auto-generated and not meant to be edited.

@theoweiss, any input?

**Comment by theoweiss** [comment]
Chris you are right this is generated code which is not meant to be edited. 
It's defined here:
https://github.com/openhab/openhab1-addons/blob/18ba124225ff4292d42be76b79a08f60aa07d501/bundles/binding/org.openhab.binding.tinkerforge/model/tinkerforge.xcore#L1327

It's obvious that the code generation did not the right thing here. But I'm not sure if the method changed by this PR is really used by the binding. @Rosi2143 It would be good to know which bug this PR should fix. Any description and/or log entries would be helpful. Is there an existing issue? If not please open an issue.

BTW: I worked on the binding some weeks ago to add new devices but was not able to reliable generate valid code. I've tried diverse solutions and workarounds and asked the XCore (this is the framework I used) developers for help but with no success. Therefore I decided to start working on the reimplementation of the binding  as openHAB 2 add-on without using XCore. Therefore only bugfixes to the openhab1 tinkerforge binding would be possible which do not require a regeneration of the model code.

TL;DR
Close this PR without merging and open an issue with a bug description.

**Comment by 9037568** [comment]
@Rosi2143 please open a new issue if you have a problem that this PR was intended to solve.
